raiden/tests/integration/network/proxies/test_token_network_registry.py

```diff
@@ -67,14 +68,17 @@ def test_token_network_registry(
                 token_address=test_token_address,
                 channel_participant_deposit_limit=TokenAmount(UINT256_MAX),
                 token_network_deposit_limit=TokenAmount(UINT256_MAX),
+                block_identifier=deploy_client.blockhash(),
             )
 
     # Register a valid token
     event_filter = token_network_registry_proxy.tokenadded_filter()
+    preblockhash = deploy_client.blockhash()
     token_network_address = token_network_registry_proxy.add_token(
         token_address=test_token_address,
         channel_participant_deposit_limit=TokenAmount(UINT256_MAX),
         token_network_deposit_limit=TokenAmount(UINT256_MAX),
+        block_identifier=preblockhash,
```

Comment:
Since `block_identifier` is used to query preconditions, it could be confusing here because we check preconditions according to our "confirmed" block which is `latest - 5`. Doing this will lead to a bad practice also used in other tests.

Reply:
Shall we use the confirmed block (latest - 5) or latest here?